packages/devtools-launchpad/src/components/LandingPage.js

```diff
@@ -134,30 +136,36 @@ const LandingPage = React.createClass({
     const isConnected = name == "Firefox"
       ? this.state.firefoxConnected
       : this.state.chromeConnected;
-    const launchButton = isConnected ? null : dom.input({
+
+    const connectedStateText = isNodeSelected ?
+      null : `Please open a tab in ${name}`;
+
+    const launchButton = isConnected ? connectedStateText : dom.input({
       type: "button",
       value: `Launch ${selectedPane}`,
       onClick: () => launchBrowser(selectedPane)
     });
+
+    const targetsContent = currentTargetsExist ? dom.header({},
+      dom.input({
+        ref: "filterInput",
+        placeholder: "Filter tabs",
+        value: filterString,
+        autoFocus: true,
+        type: "search",
+        onChange: e => this.onFilterChange(e.target.value),
+        onKeyDown: e => {
+          if (targets.size === 1 && e.keyCode === 13) {
+            this.onTabClick(targets.first(), paramName);
+          }
+        }
+      })
+    ) : dom.div({ className: "hero" }, launchButton);
```

Comment:
Node is a bit tricky because we don't launch it in the same way. Lets just leave the button out and have a message like. 

> Node is not running. Try running a script with the `--inspect` param.